Presentation.Web/app/utility/toastTranslator.ts

```diff
@@ -0,0 +1,21 @@
+﻿module Kitos.Utility {
+    export class ToastTranslatorTool {
+        static translateItSystemDeletionConflictResponse(response: string): string {
+            var errorMsg = Models.ItSystem.SystemDeleteMessages;
+            switch (response) {
```

Comment:
ToUpperCase eller ToLowerCase og så slipper du for casing fejl hvis du retter dine case labels til.